### ISSUE TITLE
Update outdated dev dependencies

### ISSUE THREAD

**Issue by nfreear** [itsravenous/google-sheets-public#4]
I'll do a pull request ...

```
xxx:google-sheets-public xxx$ npm outdated
Package  Current  Wanted  Latest  Location
jest      25.1.0  25.5.4  26.1.0  @itsravenous/google-sheets-public
nock      11.7.2  11.9.1  13.0.2  @itsravenous/google-sheets-public
```

---

**Comment by yogaraptor** [comment]
Yes please! 🙌

**Comment by yogaraptor** [comment]
Fixed by #7
